src/main.rs

```diff
@@ -409,21 +430,14 @@ fn load_and_process_opds() -> Result<(), Error> {
             }
 
             let doc_path = result.save_path;
-            if doc_path.exists() {
```

Comment:
I don't think this check does anything? Even on master there was already a check earlier when queuing up `EntryResult` to be downloaded. Unless we're expecting the disk to change from under us this should never be hit unless the file already doesn't exist. I removed it here to avoid the need to propagate and repeat the last modified check.